modules/S22_php_check.sh

```diff
@@ -74,7 +82,70 @@ s22_script_check() {
     else
       COMMON_FILES_FOUND=""
     fi
-    print_output "[+] Found ""$ORANGE""parsing issues""$GREEN"" in script ""$COMMON_FILES_FOUND"":""$NC"" ""$(print_path "$LINE")" "" "$PHP_LOG"
+    print_output "[+] Found ""$ORANGE""parsing issues""$GREEN"" in script ""$COMMON_FILES_FOUND"":""$NC"" ""$(print_path "$LINE")"
     echo "$VULNS" >> "$TMP_DIR"/S22_VULNS.tmp
   fi
 }
+
+add_recommendations(){
+   local VALUE
+   local KEY
+   VALUE="$1"
+   KEY="$2"
+
+   if [[ $VALUE == *"M"* ]]; then
+      LIMIT="${VALUE//M/}"
+   fi
+
+   if [[ $KEY == *"memory_limit"* ]] && [[ $(( LIMIT)) -gt 50 ]]; then
+     return 1
+   elif [[ $KEY == *"post_max_size"* ]] && [[ $(( LIMIT)) -gt 20 ]]; then
+     return 1
+   elif [[ $KEY == *"max_execution_time"* ]] && [[ $(( LIMIT )) -gt 60 ]]; then
+     return 1
+   else
+     return 0
+   fi
+}
+
+s22_check_php_ini(){
+  mapfile -t PHP_CONF_FILES < <( find "$FIRMWARE_PATH" -xdev "${EXCL_FIND[@]}" -iname 'php.ini' -exec md5sum {} \; 2>/dev/null | sort -u -k1,1 | cut -d\  -f3 )
+  for PHP_FILE in "${PHP_CONF_FILES[@]}" ;  do
+    print_output "[*] iniscan check of ""$(print_path "$PHP_FILE")"
+    mapfile -t INISCAN_RESULT < <( "$EXT_DIR"/iniscan/vendor/bin/iniscan scan --path="$PHP_FILE" )
+    for LINE in "${INISCAN_RESULT[@]}" ; do  
+      local LIMIT_CHECK
```

Comment:
Could you include something like the following to remove all the "PASS" overhead:

```
      if [[ "$LINE" == "PASS"* ]]; then
        continue
      fi
```

Reply:
I have removed:```
 elif ( echo "$LINE" | grep -q "PASS" ) ; then
        print_output "$(green "$LINE")
```"
It should have the same effect

Reply:
less code is better ;)